### DISCUSSION TITLE
any way to add events here?

### DISCUSSION THREAD

**Discussion by AndroXStudiO** [Sketchware-Pro/Sketchware-Pro#573] [Q&A] [answered]
i want to add onTaskRemoved() event here

![Screenshot_2022-09-06-22-12-28-64_423e58f25c30a02453fa5d8c82fc37f2](https://user-images.githubusercontent.com/88816748/188684885-a676a086-7fc1-4344-8b43-16721d0e4460.jpg)

**Comment by j-px** [comment, marked as answer]
There's Custom Events. And also a tutorial on them in our Discord server in channel #tips-n-tricks.

**Reply by AndroXStudiO** [reply]
Thanks
